R/desc.R

```diff
@@ -30,7 +34,8 @@ fill_desc <- function(
   author_first_name = NULL,
   author_last_name = NULL,
   author_email = NULL,
-  author_orcid = NULL
+  author_orcid = NULL,
+  set_options = FALSE
```

Comment:
I would love this to be set to TRUE by default :)

Reply:
done, but I keep the versions in `01_start.R` and `a_start.Rmd` files to `FALSE`, or?

Reply:
No, let's put it to TRUE everywhere :) The idea with the merge of the two function is to be sure that this fun is run, so it's better if it's TRUE by default. Thanks !

Reply:
Changes commited, thanks :)